src/hooks/tradingView/useTradingView.ts

```diff
@@ -23,14 +25,19 @@ import { getSavedResolution, getWidgetOptions, getWidgetOverrides } from '@/lib/
  */
 export const useTradingView = ({
   tvWidgetRef,
+  displayButtonRef,
   setIsChartReady,
 }: {
   tvWidgetRef: React.MutableRefObject<any>;
+  displayButtonRef: React.MutableRefObject<any>;
   setIsChartReady: React.Dispatch<React.SetStateAction<boolean>>;
```

Comment:
is there a type we could use to replace `any` here, and maybe we can update the tvWidgetRef type as well.

Reply:
addressed [here](https://github.com/dydxprotocol/v4-web/pull/292/files#diff-c8b3ff077660b9f0fa861c122ef4a719a9d312fd37ac918cdadca357f568f285R33) in stacked PR